p2p/src/interface/types.rs

```diff
@@ -26,4 +27,13 @@ pub struct ConnectedPeer {
     pub inbound: bool,
 
     pub ban_score: u32,
+
+    /// Time spent waiting for a current ping response
+    pub ping_wait: Option<RpcTime>,
```

Comment:
Same... I don't understand why a "wait" is measured in absolute time... this is all over the place.

Reply:
Should be OK now.